Tiltfile

```diff
@@ -113,6 +113,25 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+addon_settings_files = []
+
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
+        "context": "./test/extension",
+        "image": "gcr.io/k8s-staging-cluster-api/test-extension",
+        "container_name": "extension",
+        "binary_name": "extension",
+        "live_reload_deps": ["main.go", "handlers"],
+        "label": "test-extension",
+        "resource_deps": ["capi_controller"],
```

Comment:
Do you know if that works? This is a reference to the label, right? Not to a specific resource.

(I had an error deploying ExtensionConfig, because the webhook was not up)

Reply:
It should be working, and it's working for me when I disable the CAPI controller 🤔 We should try to figure out where this fails though.

Reply:
Okay sounds good. I just tried it with tilt up and then the test-extension couldn't be deployed. Let's follow-up on that